static/js/publisher/builds/index.js

```diff
@@ -15,6 +21,8 @@ class Builds extends React.Component {
     this.fetchTimer = null;
 
     this.state = {
+      triggerBuildLoading: false,
+      triggerBuildSucccess: false,
```

Comment:
as this can be `ERROR`, `SUCCESS` and `IDLE` now the initial state should probably be `IDLE` and the name of the variable would be better as `triggerBuildStatus`.